src/call_reply.c

```diff
@@ -525,3 +525,15 @@ CallReply *callReplyCreate(sds reply, list *deferred_error_list, void *private_d
     res->deferred_error_list = deferred_error_list;
     return res;
 }
+
+/* Create a new CallReply struct from the reply blob representing an error message.
+ * Automatically creating deferred_error_list and set a copy of the reply in it.
+ * Refer to callReplyCreate for detailed explanation. */
+CallReply *callReplyCreateError(sds reply, void *private_data) {
+    sds err_buff = sdscatfmt(sdsempty(), "-%S\r\n", reply);
```

Comment:
maybe we should only add `-` (actually `-ERR`) if missing?
and modify the callers by either removing the `ERR`, or adding `-` in all of them?

Reply:
Fixed